.github/workflows/unittests.yml

```diff
@@ -41,12 +41,12 @@ jobs:
       id: check_file_changed
       run: |
         $diff = git diff --name-only ${{ github.event.pull_request.base.sha }} ${{ github.sha }}
-        $SourceDiff = $diff | Where-Object { $_ -match '^content/' -or $_ -match '^grammars/' -or $_ -match '^highlighting/' }
+        $SourceDiff = $diff | Where-Object { $_ -match '^content/' -or $_ -match '^grammars/' -or $_ -match '^highlighting/' -or $_ -match '^tests/' -or $_ -match '^app.py' -or $_ -match '^hedy.py'}
         $HasDiff = $SourceDiff.Length -gt 0
         Write-Host "::set-output name=docs_changed::$HasDiff"
     - name: Run all tests
       shell: pwsh
-      if: steps.check_file_changed.outputs.docs_changed == 'True'
+      if: steps.check_file_changed.outputs.docs_changed == 'True' -or ${{ github.event.label.name == 'force_tests' }}
```

Comment:
Ah cool, I did not know this was possible!

Reply:
Not really sure that it works, but according to Stackoverflow it should...